vignettes/usage.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Usage"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Usage}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(laminr)
+```
+
+## Instance
+
+### Connect to a LaminDB instance
+
+```{r connect}
+db <- connect("laminlabs/cellxgene")
+```
+
+
+### Print a LaminDB instance
+
+```{r print_instance}
+db
+```
+
+### Get module
+
```

Comment:
Maybe each object type could have a one line description and a link to the Python docs?